### PR TITLE
Fix grammar on exceptions

### PR DESCRIPTION
**Pull request by jakesyl**

### CODE REVIEW

File: src/Knp/Component/Pager/Paginator.php [hunk 1]
```diff
@@ -84,7 +84,7 @@ public function paginate($target, $page = 1, $limit = 10, array $options = array
     {
         $limit = intval(abs($limit));
         if (!$limit) {
-            throw new \LogicException("Invalid item per page number, must be a positive number");
+            throw new \LogicException("Invalid item per page number.  limit must be a positive number");
```
Comment (NiR-) [inline src/Knp/Component/Pager/Paginator.php]: I'm afraid there's an extra space here (or a missing $).

File: src/Knp/Component/Pager/Paginator.php [hunk 2]
```diff
@@ -112,15 +112,15 @@ public function paginate($target, $page = 1, $limit = 10, array $options = array
         $itemsEvent->target = &$target;
         $this->eventDispatcher->dispatch('knp_pager.items', $itemsEvent);
         if (!$itemsEvent->isPropagationStopped()) {
-            throw new \RuntimeException('One of listeners must count and slice given target');
+            throw new \RuntimeException('One of the listeners must count and slice a given target');
```
Comment (NiR-) [inline src/Knp/Component/Pager/Paginator.php]: Wouldn't it worth it to add a little note about stopping event propagation?

File: src/Knp/Component/Pager/Paginator.php [hunk 3]
```diff
@@ -112,15 +112,15 @@ public function paginate($target, $page = 1, $limit = 10, array $options = array
...
         }
         // pagination initialization event
         $paginationEvent = new Event\PaginationEvent;
         $paginationEvent->target = &$target;
         $paginationEvent->options = &$options;
         $this->eventDispatcher->dispatch('knp_pager.pagination', $paginationEvent);
         if (!$paginationEvent->isPropagationStopped()) {
-            throw new \RuntimeException('One of listeners must create pagination view');
+            throw new \RuntimeException('One of the listeners must create pagination view');
```
Comment (NiR-) [inline src/Knp/Component/Pager/Paginator.php]: Same here

Comment (jakesyl) [inline src/Knp/Component/Pager/Paginator.php]: Got it.  I'll fix that now.

**Comment by jakesyl** [comment]
@NiR- please see my updates.

File: src/Knp/Component/Pager/Paginator.php [hunk 4]
```diff
@@ -112,15 +112,15 @@ public function paginate($target, $page = 1, $limit = 10, array $options = array
         $itemsEvent->target = &$target;
         $this->eventDispatcher->dispatch('knp_pager.items', $itemsEvent);
         if (!$itemsEvent->isPropagationStopped()) {
-            throw new \RuntimeException('One of listeners must count and slice given target');
+            throw new \RuntimeException('One of the listeners must count and slice a given target. If this does not happen propagation is stopped.');
```
Comment (NiR-) [inline src/Knp/Component/Pager/Paginator.php]: Actually, it's quite the opposite: it fails if the event propagation isn't stopped. Thus you would rather say that: `one of the listener must count and slice a target, and stop event propagation.`, or something like that.

**Comment by NiR-** [comment]
Btw, you need to fix tests: txo of them match the exception messages.

**Comment by jakesyl** [comment]
I see the errors in the files on travis ci. Where do these assertions actually take place?

**Pull request closed without merging**